mender-update/daemon/state_machine.cpp

```diff
@@ -280,6 +285,16 @@ error::Error StateMachine::Run() {
 	runner_.PostEvent(StateEvent::InventoryPollingTriggered);
 	runner_.PostEvent(StateEvent::DeploymentPollingTriggered);
 
+	// Register signal handlers
+	check_update_handler_.RegisterHandler({SIGUSR1}, [this](events::SignalNumber signum) {
+		runner_.PostEvent(StateEvent::DeploymentPollingTriggered);
+	});
+	inventory_update_handler_.RegisterHandler({SIGUSR2}, [this](events::SignalNumber signum) {
+		runner_.PostEvent(StateEvent::InventoryPollingTriggered);
+	});
+	termination_handler_.RegisterHandler(
+		{SIGTERM, SIGINT, SIGQUIT}, [this](events::SignalNumber signum) { event_loop_.Stop(); });
```

Comment:
Can you move this into its own method, and put that inside a `platform` file? First of all, not all the signals are actually [present in the C++ standard](https://en.cppreference.com/w/cpp/header/csignal), and just intuitively, we know this isn't going to be used on embedded platforms. 🙂